packages/opsian/opsian.0.1/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+synopsis: "Low overhead profiling"
+maintainer: "Opsian <admin@opsian.com>"
+authors:
+  "Richard Warburton <richard@opsian.com>, Sadiq Jaffer <sadiq@opsian.com>"
+license: "MIT"
+homepage: "https://github.com/Opsian/opsian-ocaml/wiki"
+bug-reports: "https://github.com/Opsian/opsian-ocaml/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "ocaml" {>= "4.12.0"}
+]
+flags: [ verbose ]
```

Comment:
Do you need to force verbose builds for this package? In general, the opam-repo prefers to let the user set this when installing.

Reply:
Will be fixed in latest push.